### ISSUE TITLE
understanding the synchronization

### ISSUE THREAD

**Issue by foxmask** [laurent22/joplin#294]
### Operating system

- [x] Windows
- [x] macOS
- [x] Linux
- [x] Android
- [x] iOS

### Application

- [x] Desktop
- [x] Mobile
- [x] Terminal

Hi,
When I migrated all my evernote notes (which is still not completed:), the "objects" that are synchonized in the "synchonization status" page, show us that the notes are sync before the folders and tags.
When we sync that "incomplete" statement, we can't read the notes that are already sync as there are not tied to any tags nor folders. 
Is it not possible to sync folders and tags first, then the notes so we could see the note that have already been sync  ?
It's just to understand ;)
Regards

**Comment by laurent22** [comment]
The synchroniser processes each item in no particular order (in the order the cloud service sends them) so notes might indeed arrive before notebooks or tags.

To do it differently would require adding metadata to the sync target, which would make things a lot more complicated, even though it won't make a huge difference. It will only be noticeable indeed during an initial sync with many items, as you need to wait for the notebooks to arrive.

So basically it's not really worth it given the cost in terms of complexity and relatively small benefits.

**Comment by foxmask** [comment]
thank you for the explanations
